content/physical_properties/electrical_conductivity/electrical_conductivity_lab_setup_measurements.rst

```diff
@@ -1,225 +1,209 @@
 .. _electrical_conductivity_lab_setup_measurements:
 
-Lab set up and measurements
-===========================
+Laboratory Measurements
+=======================
 
-Electrical conductivity can be frequency-dependent and complex-valued, which
-effectively generates polarization effects with the applied electric field.
-This is often called induced polarization (IP) effects. In order to measure
-complex conductivity of a rock specimen, we inject sinusoidal (that is, AC
-currents) in to the rock, and measure voltages. :numref:`GSCcompleximpedance`
-shows the lab set-up at Geological Survey of Canada (GSC). The frequency range
-of the lab set-up is from 0.016 Hz - 1 MHz (logarithmically-sampled).
+Electrical conductivity/resistivity measurements are an integral part of classifying physical rock properties. Here, we present the general approach for measuring a rock’s electrical properties, including electrical conductivity/resistivity and chargeability. Instrumentation is also discussed.
 
-.. figure:: ./images/GSCcompleximpedance.png
-   :figwidth: 50%
+Two-Electrode Laboratory Setup
+------------------------------
+
+Electrical resistivity measurements are most commonly performed using a two-electrode setup. This is illustrated in :numref:`fig_cond_lab_setup`. For these measurements, a rock core of known dimensions is placed between two copper or graphite electrodes. Current is then forced through the rock, which acts as the resistive element for an electrical circuit. By measuring the voltage drop across the rock, Ohm's law can be used to determine the corresponding resistance. Given that we know the dimensions of the rock sample, Pouillet's law can be used to obtain the rock's electrical resistivity. The electrical conductivity is obtained by taking the reciprocal of the resistivity.
```

Comment:
I not commonly seen the relationship between electrical resistance and resistivity named (`Pouillet’s law`). Here, I don't think that is adding much, instead, I would recommend using the equation reference (or section reference)